lib/hg-utils.coffee

```diff
@@ -291,7 +292,10 @@ class Repository
         if err
           reject err
         if stderr?.length > 0
-          reject stderr
+          if isWindows and stderr.length > windowsMaxPathLength
+            resolve(stdout)
```

Comment:
What's the error thrown in this case? The result still comes from the stdout? Just files with path < 260?

If I'm right, maybe we need to put a comment here specifying that the error is probably something related to a big file path. 

Whats leads us to: is there anything in the error message that could specify it better than its length being greater than 260 chars?